### DISCUSSION TITLE
Update camera eye using set_world_pose_from_view

### DISCUSSION THREAD

**Discussion by balaprenithreddy** [isaac-sim/IsaacLab#829] [Q&A]
Hello,

I implemented a USD camera sensor using the sensor listed in omni.isaac.lab.sensors and attached it to a cube. I wanted the camera to always look at the target, so I used the function set_world_pose_from_view from the camera class to look at the target in the get_observation function. When I tested my env with a random agent, sometimes I see only the target but most of the times I see my cube as well in the camera. I am also updating the eye position using the cube root_pos_w and am setting the offset to position my camera above the cube.

Currently, I am confused and not sure what i missed or am doing something wrong.

Any help is appreciated.

**Comment by Dhoeller19** [comment]
Hi @balaprenithreddy,
I believe that this might be due to rendering frame delays, which has been fixed in the latest release 1.2.0 with Isaac Sim 4.2.0. 
Could you try this out again?
